x-pack/metricbeat/module/activemq/queue/manifest.yml

```diff
@@ -35,3 +35,13 @@ input:
         field: messages.enqueue.time.min
       - attr: ProducerCount
         field: producers.count
+processors:
+  - script:
+      lang: javascript
+      source: >
+        function process(event) {
+          var queue_memory_broker_pct = event.Get("activemq.queue.memory.broker.pct")
+          if (queue_memory_broker_pct != null) {
+            event.Put("activemq.queue.memory.broker.pct", queue_memory_broker_pct / 100.0)
```

Comment:
I see we have an assertion in `test_activemq.py` checking that this field was exist. How did it work before if `get_activemq_module_config` generated a config without processors?

Reply:
The field was always available, no matter if processors are up or down. The script processor only adjusts its value (pct) to be in range 0..1 instead 0..100.

Reply:
Oh true :sweat_smile: Wdyt about adding an assertion to check that the value is in the range 0..1 to check that the processors are being applied?

Reply:
According to the offline discussion, the processors setup will be verified using IBM MQ system tests.
I updated the IBM MQ system test to verify if all metrics have the same prefix (`ibmmq_`). It will be sufficient as all metrics that are emitted at first are stricly related to Go runtime (e.g. `go_gc_*`). The broker starts after a couple of seconds.

You can also in metricbeat.log following entries:

```
2020-01-31T12:30:39.138+0100	DEBUG	[publisher]	pipeline/client.go:220	Pipeline client receives callback 'onFilteredOut' for event: {Timestamp:2020-01-31 12:30:39.13213 +0100 CET m=+0.043040558 Meta:null Fields:{"event":{"dataset":"ibmmq.qmgr","duration":5780570,"module":"ibmmq"},"metricset":{"name":"qmgr","period":5000},"prometheus":{"labels":{"instance":"localhost:9157","job":"ibmmq"},"metrics":{}},"service":{"address":"localhost:9157","type":"ibmmq"}} Private:<nil> TimeSeries:true}
2020-01-31T12:30:39.138+0100	DEBUG	[publisher]	pipeline/client.go:220	Pipeline client receives callback 'onFilteredOut' for event: {Timestamp:2020-01-31 12:30:39.13213 +0100 CET m=+0.043040558 Meta:null Fields:{"event":{"dataset":"ibmmq.qmgr","duration":5788148,"module":"ibmmq"},"metricset":{"name":"qmgr","period":5000},"prometheus":{"labels":{"handler":"prometheus","instance":"localhost:9157","job":"ibmmq"},"metrics":{}},"service":{"address":"localhost:9157","type":"ibmmq"}} Private:<nil> TimeSeries:true}
2020-01-31T12:30:39.138+0100	DEBUG	[publisher]	pipeline/client.go:220	Pipeline client receives callback 'onFilteredOut' for event: {Timestamp:2020-01-31 12:30:39.13213 +0100 CET m=+0.043040558 Meta:null Fields:{"event":{"dataset":"ibmmq.qmgr","duration":5790490,"module":"ibmmq"},"metricset":{"name":"qmgr","period":5000},"prometheus":{"labels":{"handler":"prometheus","instance":"localhost:9157","job":"ibmmq","quantile":"0.5"},"metrics":{}},"service":{"address":"localhost:9157","type":"ibmmq"}} Private:<nil> TimeSeries:true}
2020-01-31T12:30:39.138+0100	DEBUG	[publisher]	pipeline/client.go:220	Pipeline client receives callback 'onFilteredOut' for event: {Timestamp:2020-01-31 12:30:39.13213 +0100 CET m=+0.043040558 Meta:null Fields:{"event":{"dataset":"ibmmq.qmgr","duration":6297712,"module":"ibmmq"},"metricset":{"name":"qmgr","period":5000},"prometheus":{"labels":{"instance":"localhost:9157","job":"ibmmq","quantile":"0"},"metrics":{}},"service":{"address":"localhost:9157","type":"ibmmq"}} Private:<nil> TimeSeries:true}
2020-01-31T12:30:39.138+0100	DEBUG	[publisher]	pipeline/client.go:220	Pipeline client receives callback 'onFilteredOut' for event: {Timestamp:2020-01-31 12:30:39.13213 +0100 CET m=+0.043040558 Meta:null Fields:{"event":{"dataset":"ibmmq.qmgr","duration":6383620,"module":"ibmmq"},"metricset":{"name":"qmgr","period":5000},"prometheus":{"labels":{"instance":"localhost:9157","job":"ibmmq","quantile":"0.25"},"metrics":{}},"service":{"address":"localhost:9157","type":"ibmmq"}} Private:<nil> TimeSeries:true}
2020-01-31T12:30:39.138+0100	DEBUG	[publisher]	pipeline/client.go:220	Pipeline client receives callback 'onFilteredOut' for event: {Timestamp:2020-01-31 12:30:39.13213 +0100 CET m=+0.043040558 Meta:null Fields:{"event":{"dataset":"ibmmq.qmgr","duration":6413962,"module":"ibmmq"},"metricset":{"name":"qmgr","period":5000},"prometheus":{"labels":{"instance":"localhost:9157","job":"ibmmq","quantile":"0.5"},"metrics":{}},"service":{"address":"localhost:9157","type":"ibmmq"}} Private:<nil> TimeSeries:true}
```